gitlab/config.json

```diff
@@ -0,0 +1,91 @@
+{
+    "display_name": "Gitlab",
+    "description": "Gitlab external plugin",
+    "logo_file": "Gitlab-Logo.png",
+    "integration_type": "ALM",
+    "is_webhook_at_project_level": true,
+    "instance_properties": [
+        {
+            "title": "Name",
+            "name": "name",
+            "description": "Name of the instance",
+            "type": "input"
+        },
+        {
+            "title": "Username",
+            "name": "Username",
```

Comment:
can you make this in lower case

Reply:
i used this on lot of places,so chnaginf it may rise a issue'